pyfr/integrators/dual/steppers.py

```diff
@@ -1,7 +1,62 @@
 # -*- coding: utf-8 -*-
 
 from pyfr.integrators.dual.base import BaseDualIntegrator
-
+from pyfr.util import memoize
 
```

Comment:
Add a newline after this import so that there are two blank lines between it and the class definition.